rtgui/main.cc

```diff
@@ -308,109 +580,51 @@ int main(int argc, char **argv)
             printf("Error: -gimp requires two arguments\n");
             return 1;
         }
+    } else if (!remote && Glib::file_test(argv1, Glib::FILE_TEST_EXISTS)) {
+        simpleEditor = true;
     }
 
-    Gtk::Main m(&argc, &argv);
-
-    Glib::ustring icon_path = Glib::build_filename(argv0, "images");
-    Glib::RefPtr<Gtk::IconTheme> defaultIconTheme = Gtk::IconTheme::get_default();
-    defaultIconTheme->append_search_path(icon_path);
-
-    rtengine::setPaths(options);
-    MyExpander::init();  // has to stay AFTER rtengine::setPaths
-
-    // ------- loading theme files
-
-    Glib::RefPtr<Gdk::Screen> screen = Gdk::Screen::get_default();
-
-    if (screen) {
-        Gtk::Settings::get_for_screen(screen)->property_gtk_theme_name() = "Adwaita";
-        Gtk::Settings::get_for_screen(screen)->property_gtk_application_prefer_dark_theme() = true;
-
-        Glib::RefPtr<Glib::Regex> regex = Glib::Regex::create(THEMEREGEXSTR, Glib::RegexCompileFlags::REGEX_CASELESS);
-        Glib::ustring filename = Glib::build_filename(argv0, "themes", options.theme + ".css");
-        if (!regex->match(options.theme + ".css") || !Glib::file_test(filename, Glib::FILE_TEST_EXISTS)) {
-            options.theme = "RawTherapee-GTK";
-            // We're not testing GTK_MAJOR_VERSION == 3 here, since this branch requires Gtk3 only
-            if (GTK_MINOR_VERSION < 20) {
-                options.theme = options.theme + "3-_19";
-            } else {
-                options.theme = options.theme + "3-20_";
-            }
-            filename = Glib::build_filename(argv0, "themes", options.theme + ".css");
-        }
-        cssRT = Gtk::CssProvider::create();
-
-        try {
-            cssRT->load_from_path (filename);
-            Gtk::StyleContext::add_provider_for_screen(screen, cssRT, GTK_STYLE_PROVIDER_PRIORITY_APPLICATION);
-        } catch (Glib::Error &err) {
-            printf("Error: Can't load css file \"%s\"\nMessage: %s\n", filename.c_str(), err.what().c_str());
-        } catch (...) {
-            printf("Error: Can't load css file \"%s\"\n", filename.c_str());
+    int ret = 0;
+    if (remote) {
+        char *app_argv[2] = { nullptr, nullptr };
```

Comment:
https://developer.gnome.org/gtkmm/stable/classGtk_1_1Application.html#a41ede16109c06bbefce5194f4b601780

So, `char **`, not `const char **`